packages/grid/backend/grid/core/node.py

```diff
@@ -16,52 +18,61 @@
 # grid absolute
 from grid.core.config import settings
 
-mongo_client_config = MongoStoreClientConfig(
-    hostname=settings.MONGO_HOST,
-    port=settings.MONGO_PORT,
-    username=settings.MONGO_USERNAME,
-    password=settings.MONGO_PASSWORD,
-)
 
-mongo_store_config = MongoStoreConfig(client_config=mongo_client_config)
+def mongo_store_config() -> MongoStoreConfig:
+    mongo_client_config = MongoStoreClientConfig(
+        hostname=settings.MONGO_HOST,
+        port=settings.MONGO_PORT,
+        username=settings.MONGO_USERNAME,
+        password=settings.MONGO_PASSWORD,
+    )
+
+    return MongoStoreConfig(client_config=mongo_client_config)
+
 
+def sql_store_config() -> SQLiteStoreConfig:
+    client_config = SQLiteStoreClientConfig(path=settings.SQLITE_PATH)
+    return SQLiteStoreConfig(client_config=client_config)
+
+
+def seaweedfs_config() -> SeaweedFSConfig:
```

Comment:
Good work, seggregating it into distinct functions , make the code more readable.